.prettierignore

```diff
@@ -1,2 +1,3 @@
 dist
-node_modules
\ No newline at end of file
+node_modules
+src
```

Comment:
We should not exclude src from prettier, I assume you might be doing this because you don't have prettier configured properly but that doesn't mean we should all stop using it! You can either disable your prettier from IDE or fix the config hehe